lib/OPCUA/Open62541/Test/Server.pm

```diff
@@ -93,6 +93,7 @@ sub child {
     my $running = 1;
     $SIG{ALRM} = $SIG{TERM} = sub { $running = 0; };
     $SIG{USR1} = sub { note("SIGUSR1 received"); };
+    $SIG{USR2} = sub { note("SIGUSR2 received"); $self->{next_action} = 1; };
```

Comment:
It is not necessary to store this bit in the server hash.  It is local to this method.  Could you add a $action = 0 like it is done for $running?